### PR TITLE
Enhance raise issues

### PR DESCRIPTION
**Pull request by cpaelzer**

### CODE REVIEW

**Comment by cpaelzer** [comment]
BTW - this might also be the chance to *standarize* the triage-mails format and content.
Let me know if you'd want that to be added here and also let me know who you think has the most beautiful and useful mails so we can use them as the initial template for discussion.

File: BugTriage.md [hunk 1]
```diff
@@ -156,7 +156,29 @@ wiki (https://wiki.ubuntu.com/ServerTeam).
...
 
 
-Raise Issue with the Team
--------------------------
+Awareness of the Triage
+-----------------------
 
-TODO: I.e. at standup meeting, or on IRC if it's urgent.
+We have several stakeholders we want to keep up-to-date on things that
+we've found on triage. n one hand we want to keep the community generally
```
Comment (utkarsh2102) [inline BugTriage.md:163]: `n` -> `On`

File: BugTriage.md [hunk 2]
```diff
@@ -156,7 +156,29 @@ wiki (https://wiki.ubuntu.com/ServerTeam).
...
+
+Furthermore on cases that need immediate attention or at least awareness
+we might:
+
+ * Bring them up in the daily standup (mostly if they need a discussion/decision that one can't do alone)
+ * Ping a subject matter expert via IRC/Mattermost
+
+In some cases a package maintainer might already be aware and follow a case.
+To avoid endless re-pings on such a case the agreement is that if the maintainer is
+personally (i.e. with his launchpad user, not just indirectly via teams like
```
Comment (utkarsh2102) [inline BugTriage.md:182]: The sentence seems incomplete. I guess you mean "personally subscribed" or something similar?

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
Thanks, Christian. Fixes requested below (inline diff).

> BTW - this might also be the chance to standarize the triage-mails format and content.

That sounds like a great idea! I am always a huge +1 for consistency.

> BTW - this might also be the chance to standarize the triage-mails format and content.
> Let me know if you'd want that to be added here and also let me know who you think has the most beautiful and useful mails so we can use them as the initial template for discussion.

I'll skim through and come up with something later today or tomorrow. Can we put in a separate PR, I guess.

File: BugTriage.md (hunk 1 above)
Comment (cpaelzer) [reply BugTriage.md:163]: Thanks - done

File: BugTriage.md (hunk 2 above)
Comment (cpaelzer) [reply BugTriage.md:182]: indeed - fixed

File: BugTriage.md [hunk 3]
```diff
@@ -156,7 +156,30 @@ wiki (https://wiki.ubuntu.com/ServerTeam).
...
+we might:
+
+ * Bring them up in the daily standup (mostly if they need a discussion/decision that one can't do alone)
+ * Ping a subject matter expert via IRC/Mattermost
+
+In some cases a package maintainer might already be aware and follow a case.
+To avoid endless re-pings on such a case the agreement is that if the maintainer
+is personally subscribed (i.e. with his launchpad user, not just indirectly via
+teams like [Ubuntu storage interest](https://launchpad.net/~ubuntu-storage))
+then we consider the maintainer to be aware and will not do extra
```
Comment (utkarsh2102) [inline BugTriage.md:184]: `s/maintainer/care taker/g` or something? But meh, super trivial 😅

**Review by utkarsh2102 — APPROVED** [review]
Super awesome, looks good! 🎉

**Pull request merged**